README.md

```diff
@@ -158,6 +160,27 @@ _signature_, you'll find that it matches the signatures below. Notice that all
 the methods returns a promise. A method with `: void` also returns a promise,
 that either resolves without giving a value or rejects with an error.
 
+## Fake Listening
+
+It's inconvenient and error-prone to use real Pulse credentials in tests.  The
+PulseListener class supports a "fake" mode where it does not use any
+credentials, and messages are delivered by means of a `fakeMessage` method.
+
+To set up the listener, intialize it with `credentials: {fake: true}`. Once the
```

Comment:
Instead of `fake: true` could we make it `mock: true` ? similar to what we do in tc-lib-monitor